doc/user/tutorials/making_your_first_app.rst

```diff
@@ -0,0 +1,510 @@
+================================
+Making your first ARMI-based App
+================================
+
+In this tutorial we will build a nuclear analysis application that runs (dummy) neutron
+flux and thermal/hydraulics calculations. Real applications that do real analysis can be
+modeled after this starting point.
+
+We'll assume you have the :doc:`ARMI Framework installed </user/user_install>` already.
+You can make sure it is ready by running the following command in a shell prompt::
+
+    (armi) $ python -c "import armi;armi.configure()"
+
+You should see an ARMI splash-screen and an ARMI version print out. If you do, you are ready
+to proceed.
+
+.. tip:: If you are having trouble getting it installed, see :ref:`getting-help`. You may
+    need to ensure your ``PYTHON_PATH`` variable includes the armi installation directory.
+
+.. note:: This tutorial is a companion for the :doc:`/developer/making_armi_based_apps`
+    developer documentation.
```

Comment:
Do we want to call out the `dragon` plugin or example armi app here?

Reply:
I think we're going to replace the example app with this particular code and then will call it out here.